.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
You should put the .vscode folder to ignore in the .git configurtaion.

Reply:
or just remove this file. Its a problem of IDE